package.json

```diff
@@ -53,6 +52,7 @@
     "typescript": "^4.1.3"
   },
   "dependencies": {
+    "@types/yargs": "^16.0.0",
```

Comment:
`@types/yargs` should stay in devDependencies, not dependencies. You might have done `npm i` when you meant to do `npm i -D`. You could just checkout this and `package-lock.json`.